styles.css

```diff
@@ -66,3 +106,28 @@ fieldset button {
   padding: 10px 30px;
   border-radius: 2px solid black;
 }
+
+/* contact section */
+
+.contact {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.contact p {
+  margin: 0;
+}
+
+/* Footer section */
+
+footer {
+  border: 2px solid black;
+  padding: 10px;
+  border-radius: 5px;
+}
```

Comment:
[ Optional Changes ]

This change won't stop your project from being approved but will greatly help make your program looks good.

- [ ] It would be nice if you could place the footer at the bottom of the page just as it is in the [wireframe](https://github.com/microverseinc/curriculum-javascript/blob/main/books/m4_full_website_v1_1.md#project-requirements) in all pages. You could use CSS positioning to position the footer at the bottom of the page. 👍🏽

<img width="1100" alt="Screenshot 2023-05-31 at 10 01 28 PM" src="https://github.com/Salarhabibi-amir/AwsomeBook/assets/50941074/f26fef05-de51-41a2-b1e4-99130c772370">

Reply:
[ Optional Changes ]

- [ ] What if you could clear the input fields after adding a book so the user doesn't have to clear them manually? This could improve the user experience of your app. 

<img width="644" alt="Screenshot 2023-05-31 at 10 00 38 PM" src="https://github.com/Salarhabibi-amir/AwsomeBook/assets/50941074/764a3c3c-f307-4ca8-bdb6-10894f74dd68">

Reply:
[ Optional Changes ]

- [ ] What if you could clear the input fields after adding a book so the user doesn't have to clear them manually? This could improve the user experience of your app. 

<img width="644" alt="Screenshot 2023-05-31 at 10 00 38 PM" src="https://github.com/Salarhabibi-amir/AwsomeBook/assets/50941074/764a3c3c-f307-4ca8-bdb6-10894f74dd68">